contracts/governance/Vesting/VestingLogic.sol

```diff
@@ -116,7 +116,9 @@ contract VestingLogic is IVesting, VestingStorage, ApprovalReceiver {
      * @dev Can be called only by owner.
      * */
     function governanceWithdrawTokens(address receiver) public {
-        revert("deprecated, use cancelTeamVesting from the staking contract");
+        require(msg.sender == address(staking), "unauthorized");
```

Comment:
missing a warning comment that this function should not be used with sovryn protocol